core/templates/dev/head/pages/admin/activities_tab/admin_dev_mode_activities_tab_directive.html

```diff
@@ -19,6 +19,19 @@
             ng-click="reloadAllExplorations()">
       Reload All Explorations
     </button>
+    <h3>Reload dummy explorations</h3>
+    <span class="col-lg-4 col-md-4 col-sm-4 row">
+      <input class="form-control" style="width:80%"
+      ng-model= "DummyExplorationCount" placeholder="No. of explorations to load">
```

Comment:
Remove space before "DummyExplorationCount".
Align left end of 'ng-model' with left end of 'class' in the line above.